CIFAR10/train/BAIDU-USA-GAIT_BaiduNet9P_8GV100_pytorch.json

```diff
@@ -0,0 +1,20 @@
+{
+	    "version": "v1.0",
+	    "author": "Baidu USA GAIT LEOPARD team: Baopu Li, Zhiyu Cheng, Yingze Bao", 
+	    "authorEmail": "baopuli@baidu.com, zhiyucheng@baidu.com, yingzebao@baidu.com",
+	    "framework": "PyTorch v1.0.1",
+	    "codeURL": "https://github.com/BAIDU-USA-GAIT-LEOPARD/CIFAR10-Distributed-Training-BaiduNet9P/commit/df05b74663f21d4cbed894590535fab9a79d4d32",
+	    "model": "BaiduNet9P",
+	    "hardware": "Baidu Cloud Tesla 8*V100-16GB/448 GB/96 CPU",
+	    "costPerHour": 8.5058,
+	    "timestamp": "2019-05-28",
+	    "misc": {
+			"1cycle parameters": "cycles: 24,cyclical max/min momentum: 0.95,0.85, pct_start = 0.41， max=0.0052",

```

Comment:
This is pretty hard to understand, but I am not sure of a better way to phrase it. I am just going to include a link to fast.ai's code for future reference:

[https://github.com/fastai/fastai/blob/79ecfd5c60087766c6f1952e674f98470d488069/fastai/callbacks/one_cycle.py](https://github.com/fastai/fastai/blob/79ecfd5c60087766c6f1952e674f98470d488069/fastai/callbacks/one_cycle.py)

Reply:
Thanks a lot for the supplementary link.